extensions/jaeger/runtime/src/main/java/io/quarkus/jaeger/runtime/JaegerConfig.java

```diff
@@ -111,4 +111,10 @@
     @ConfigItem
     public Optional<String> senderFactory;
 
+    /**
+     * Whether the trace context should be logged.
+     */
+    @ConfigItem(defaultValue = "true")
+    public Optional<Boolean> logTraceContext;
```

Comment:
You don't need to use `Optional` here if you have a defaultValue set in the @ConfigItem